### ISSUE TITLE
why got loss = nan 

### ISSUE THREAD

**Issue by leichangqing** [ZJULearning/pixel_link#36]
why got loss =nan after some steps.

INFO:tensorflow:global step 98468: loss = nan (0.416 sec/step)

**Comment by GodOfSmallThings** [comment]
How many batches are you using? If small, try to use lower learning rate.

**Comment by GodOfSmallThings** [comment]
@changkaizhao @leichangqing  
How did you changed the model?

**Comment by changkaizhao** [comment]
nothing changed! but it now goes well. weird!!!

**Comment by changkaizhao** [comment]
@GodOfSmallThings  model params only save on `/GPU:0`,  your devices should include `/GPU:0` to train your model.

**Comment by zxDeepDiver** [comment]
I also met nan sometime.

**Comment by HuayueZhang** [comment]
> i also met nan with batch_size 32 and learning rate 0.001

There are two stages of training, the first one has 100 steps and then run the second one. Both learning rates are supposed to be modified by you. Maybe your second learning rate is too large.

**Comment by alpace** [comment]
> > i also met nan with batch_size 32 and learning rate 0.001
> 
> There are two stages of training, the first one has 100 steps and then run the second one. Both learning rates are supposed to be modified by you. Maybe your second learning rate is too large.

why need two stages of training and there is no number_of_steps of the secend.

**Comment by v-smwang** [comment]
+1

**Comment by longnanxi** [comment]
> > > i also met nan with batch_size 32 and learning rate 0.001
> > 
> > 
> > There are two stages of training, the first one has 100 steps and then run the second one. Both learning rates are supposed to be modified by you. Maybe your second learning rate is too large.
> 
> why need two stages of training and there is no number_of_steps of the secend.
hi, Has this problem been solved? I also meet this question
